internal/cli/cli.go

```diff
@@ -38,22 +39,22 @@ const (
 type config struct {
 	InstallID     string            `json:"install_id,omitempty"`
 	DefaultTenant string            `json:"default_tenant"`
-	Tenants       map[string]tenant `json:"tenants"`
+	Tenants       map[string]Tenant `json:"tenants"`
 }
 
-// tenant is the cli's concept of an auth0 tenant. The fields are tailor fit
-// specifically for interacting with the management API.
-type tenant struct {
+// Tenant is the cli's concept of an auth0 tenant.
+// The fields are tailor fit specifically for
+// interacting with the management API.
+type Tenant struct {
```

Comment:
The RunLogin cmd returns a Tenant and we shouldn't return private structs on public funcs.